bin/runsmoketest.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ue
+
+set -x
+
+
+function shutdownDocker() {
+  docker-compose -f docker-compose.smoke-tests.yml down
+}
+
+
+trap shutdownDocker INT TERM QUIT EXIT
+
+docker-compose --version
+
+
+docker-compose -f docker-compose.smoke-tests.yml run syatests
```

Comment:
Will it run properly without `docker-compose -f docker-compose.smoke-tests.yml build` first?

Reply:
We are not building docker image here, as we using existing image to reform/codeceptjs to run our tests against deployed instance